thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiFragment.java

```diff
@@ -148,9 +153,26 @@ public View onCreateView(final LayoutInflater inflater, @Nullable final ViewGrou
         return super.onCreateView(inflater, container, savedInstanceState);
     }
 
+
     @Override
     public void onDestroy() {
         super.onDestroy();
+        //FIXME handle attach/detach state
+
+        final TiConfiguration config = mPresenter.getConfig();
+
+        if (DEBUG) {
```

Comment:
[This](https://github.com/grandcentrix/ThirtyInch/pull/35#discussion_r86499780) is still valid :)
Don't put all the `Log`s  into the method. Create a own method instead which will log all the stuff.